span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanPreProcessor.java

```diff
@@ -94,6 +97,14 @@ PreProcessedSpan preProcessSpan(Span span) {
     String tenantId = maybeTenantId.get();
 
     if (spanFilter.shouldDropSpan(span, tags)) {
+      // increment dropped counter at tenant level
+      tenantToSpansDroppedCount
```

Comment:
wondering if we should just replace the metric reported for the same case at line 63?

Reply:
The ones reported at L55 and L63 are aggregate for all tenants (dropped and received counters). If we want to replace them at tenant level, then both should be done so that one could correlate them.

Reply:
We can go ahead with the changes of reporting dropped span at tenant level as well, if needed the same can be added for received spans